inst/css/filter-panel.css

```diff
@@ -218,3 +218,90 @@ a.remove_all:hover {
 .teal_active_summary_filter_panel {
   overflow-x: auto;
 }
+
+/* 
+below is the css for the new fitler card design
+will need to remove relevant code above
+*/
+
+.filter-card-container {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
+
+/* label for MAE data sets*/
+.filter-card-container[data-label]:not([data-label=""]):before {
+  padding: 2px 4px;
+  font-size: 90%;
+  color: #c7254e;
+  background-color: #f9f2f4;
+  border-radius: 4px;
+  font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+
+  display: inline-block;
+  max-width: 100%;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-weight: 700;
+
+  content: attr(data-label);
+}
+
+.filter-card {
+  width: 100%;
+  display: grid;
+  grid-template-rows: 10% 90%;
+}
+
+.filter-card-header {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  color: var(--bs-body-color, #333333);
+  border: 1px solid var(--bs-gray-200, rgba(0, 0, 0, 0.03));
+  padding: 5px 10px;
+  border-radius: 0.25rem;
+}
+
+.filter-card-header:hover {
```

Comment:
```suggestion
.filter-card:hover {
```

On hover, I can see the header changes.

![Screen Shot 2023-02-06 at 2 04 19 PM](https://user-images.githubusercontent.com/8597300/217100582-260b488e-2e01-4070-86b3-371b863c4a65.jpg)

What do you think about applying this to the filter-card instead?
![image](https://user-images.githubusercontent.com/8597300/217100996-5dbc3a2c-f417-4068-a890-0694705cca92.png)

Reply:
Good call. I'll look into this. Your screenshots look odd to me though because the hover color is supposed to apply to the title and summary. The idea is to focus on a card when it is collapsed. May be better on the whole card.

Reply:
I looked at coloring the whole card on hover. This made it hard to separate the header from the body on an expanded card. I've added hover color for the body on an expanded card. This visually separates the header and body better. Please let me know what you think about this.